app/controllers/work_packages/bulk_controller.rb

```diff
@@ -108,6 +108,17 @@ def destroy_work_packages(work_packages)
     end
   end
 
+  def possible_assignees
+    scope = Principal
+            .where(id: Principal.possible_assignee(@projects[0]))
+
+    @projects[1..].each do |project|
+      scope = scope.or(Principal.where(id: Principal.possible_assignee(project)))
+    end
```

Comment:
This reads a bit complicated, wouldn't `scope = Principal.empty` with the loop over all projects work?

Reply:
Thanks for the tip, @oliverguenther. Simplified it and actually fixed a bug in there while doing it. This was creating a union over all the possible assignees when it should have been the intersection.